### DISCUSSION TITLE
GIPO Pins ready for use on version 5.1 and later?

### DISCUSSION THREAD

**Discussion by ScytheandGenesisGames** [BlitterStudio/amiberry#998] [General]
Since GIPO Leds can be used on Amiberry 5.1 - so I still need to do that GIPO stuff then Sudo (run as root) for the program or I could just use the GPIO Pins without the need for Sudo command for Amiberry?

I just hope I can use them without that according to a guide from #129 on how to get it ready to make Amiberry use the pins as indicators for the emulator.

**Comment by StmLord** [comment]
I think it only depends on the underlying OS. If for example raspbian can give access to GPIO on non super user then you can use them without sudo on amiberry

**Reply by ScytheandGenesisGames** [reply]
Setting it up is gonna be breaking a few things with the CLI and that's something I may not do, I could mess something up and can't recover from it.

**Reply by midwan** [reply]
Did you check the wiki? https://github.com/BlitterStudio/amiberry/wiki/Using-GPIO-pins-for-LEDs
You can either run Amiberry as `sudo` or give your user access to the relevant groups, to have permissions to the GPIO pins.

**Reply by ScytheandGenesisGames** [reply]
I don't wanna break anything on the pi once I gave the Sudo to the launcher it changed the audio to I can't use HDMI as a sound input, no matter what I tried it doesn't work through hdmi.

I put Sudo in my launcher and sound is working it just changed it to SDL2: bcm2835 HDMI 1, bm2835 HDMI 1 - this is what I use for sound output
